callbacks/review_callbacks.py

```diff
@@ -83,12 +85,21 @@ def update_review_status_and_return(value, status, selected_rows):
           for row in selected_rows
       ]
       try:
-          if (status == "applied_tag"):
-            updates_dict = defaultdict(list)
-            for row in selected_rows:
-                updates_dict[row[const.SUMMARY_COLUMN_NAME_KEY]].append(row[const.SUMMARY_PII_ENTITY_KEY])
-            # TODO: Replace this with a service call once the service endpoint is ready
-            spark_client.apply_tags(value, updates_dict)
+          updates_list = []
+          for row in selected_rows:
+            updates_list.append({
+                "column_name": row[const.SUMMARY_COLUMN_NAME_KEY],
+                "data_class": row[const.SUMMARY_PII_ENTITY_KEY]
+            })
+
+          if status == "applied_tag":
+            accepted_classifications = updates_list
+            rejected_classifications = None
+          else:
```

Comment:
is the assumption here that if this callback is invoked, and the status != `applied_tag`, then it was a reject click? and does this assumption hold if we introduce another button in the future?